plugin/src/main/kotlin/org/jetbrains/research/pynose/plugin/quickfixes/unittest/SuboptimalAssertionTestSmellQuickFix.kt

```diff
@@ -0,0 +1,73 @@
+package org.jetbrains.research.pynose.plugin.quickfixes.unittest
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.util.descendants
+import com.jetbrains.python.psi.LanguageLevel
+import com.jetbrains.python.psi.PyCallExpression
+import com.jetbrains.python.psi.PyElementGenerator
+import com.jetbrains.python.psi.impl.PyExpressionStatementImpl
+import org.jetbrains.research.pynose.plugin.util.TestSmellBundle
+
+
+class SuboptimalAssertionTestSmellQuickFix : LocalQuickFix {
+
+    // todo: specify?
+    override fun getFamilyName(): String {
+        return TestSmellBundle.message("quickfixes.suboptimal.message")
+    }
+
+    private val relationTypes = mapOf(
```

Comment:
Can you remind me, why do you need this map? Why don't you just put these constants right below in the code? Or keep it as Enum, at least?